lib/kramdown-rfc2629.rb

```diff
@@ -494,19 +499,24 @@ def convert_img(el, indent, opts) # misuse the tag!
         if a = el.attr
           alt = a.delete('alt').strip
           alt = '' if alt == '!' # work around re-wrap uglyness
-          if anchor = a.delete('src')
-            a['target'] = anchor
+          if src = a.delete('src')
+            a['target'] = src
           end
         end
         if alt == ":include:"   # Really bad misuse of tag...
+          anchor = el.attr.delete('anchor')
+          if !anchor
+            warn "*** missing anchor for '#{src}'"
+            anchor = src
+          end
           to_insert = ""
-          anchor.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z_0-9.-]+)/) do |t, n|
+          src.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z_0-9.\/\+-]+)/) do |t, n|
             fn = "reference.#{t}.#{n}.xml"
             sub, ttl = XML_RESOURCE_ORG_MAP[t]
             ttl ||= KRAMDOWN_REFCACHETTL  # everything but RFCs might change a lot
             puts "Huh: ${fn}" unless sub
             url = "#{XML_RESOURCE_ORG_PREFIX}/#{sub}/#{fn}"
-            to_insert = get_and_cache_resource(url, ttl)
+            to_insert = get_and_cache_resource(url, fn.gsub('/', '_').gsub('+', '_'), ttl)
```

Comment:
I've not had good luck with odd characters in filenames and it's one that causes issues in shells and URLs. I can revert it if you like.